nock/nock-tests.ts

```diff
@@ -22,6 +22,9 @@ inst = scope.head(str);
 inst = scope.get(str);
 inst = scope.get(str, data);
 
+inst = scope.options(str);
+inst = scope.options(str, data);
```

Comment:
The document you pointed to shows `options` can take three parameters, can you change accordingly? And add a test case.